common/openapi.yml

```diff
@@ -90,6 +90,28 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/oidc/v1/{provider}/access-token:
```

Comment:
I think we should add this as an `api/oauth/v1` endpoint instead since Github is not an oidc provider

Reply:
that's how I had it originally, but Trevyn was arguing that it belongs more under `oidc` because it's not really part of the oauth flow (i think that was the reasoning, it was in an oudated comment)